symplyphysics/laws/thermodynamics/free_energy_differential.py

```diff
@@ -46,7 +51,57 @@
     -1 * entropy * temperature_change - pressure * volume_change + chemical_potential * particle_count_change,
 )
 
-# TODO: derive from the definition of Helmholtz free energy and internal energy differential
+# Derive from the definition of Helmholtz free energy and internal energy differential
```

Comment:
I suppose we need some better machinery to work with differentials, but for now, I guess, this suffices.